pkg/printers/internalversion/printers.go

```diff
@@ -1884,3 +1894,58 @@ func printControllerRevisionList(list *apps.ControllerRevisionList, options prin
 	}
 	return rows, nil
 }
+
+func printResourceQuota(resourceQuota *api.ResourceQuota, options printers.PrintOptions) ([]metav1beta1.TableRow, error) {
+	row := metav1beta1.TableRow{
+		Object: runtime.RawExtension{Object: resourceQuota},
+	}
+
+	resources := make([]api.ResourceName, 0, len(resourceQuota.Status.Hard))
+	for resource := range resourceQuota.Status.Hard {
+		resources = append(resources, resource)
+	}
+	sort.Sort(SortableResourceNames(resources))
+
+	var requestColumn string
+	var limitColumn string
+	for i := range resources {
+		resource := resources[i]
+		usedQuantity := resourceQuota.Status.Used[resource]
+		hardQuantity := resourceQuota.Status.Hard[resource]
+
+		if strings.HasPrefix(resource.String(), "request") {
+			if strings.HasPrefix(resource.String(), "requests.cpu") {
+				requestColumn += "cpu: " + usedQuantity.String() + "/" + hardQuantity.String() + ", "
+			} else {
+				requestColumn += "memory: " + usedQuantity.String() + "/" + hardQuantity.String()
+			}
+		}
+
+		if strings.HasPrefix(resource.String(), "limit") {
+			if strings.HasPrefix(resource.String(), "limits.cpu") {
+				limitColumn += "cpu: " + usedQuantity.String() + "/" + hardQuantity.String() + ", "
+			} else {
+				limitColumn += "memory: " + usedQuantity.String() + "/" + hardQuantity.String()
+			}
+		}
+	}
+
+	age := translateTimestampSince(resourceQuota.CreationTimestamp)
+	row.Cells = append(row.Cells, resourceQuota.Name, age)
+	if options.Wide {
```

Comment:
No need to place the other two cells under `-o wide`. We're only showing 4 cells total anyway